contrib/lang/erlang-elixir/packages.el

```diff
@@ -21,6 +22,41 @@
   "List of all packages to install and/or initialize. Built-in packages
 which require an initialization must be listed explicitly in the list.")
 
+(defun erlang-elixir/init-alchemist ()
+  (add-hook 'elixir-mode-hook 'alchemist-mode)
+  (setq alchemist-project-compile-when-needed t)
+  (evil-leader/set-key-for-mode 'elixir-mode
+    "mmd" 'alchemist-mix-execute
+    "mta" 'alchemist-mix-test
+    "mtb" 'alchemist-mix-test-this-buffer
+    "mtf" 'alchemist-mix-test-file
```

Comment:
`'alchemist-mix-test-file` prompts for a file to test, but you are right about removing it because it doesn't seem useful enough to have it bound! I'll also remove the other one.